content/docs/enterprise/external-data/ip-ranges.mdx

```diff
@@ -91,11 +91,13 @@ Create an external data source:
 
 1. Fill out the following fields:
 
-| Field       | Content                      | Notes                                                                                                |
-| :---------- | :--------------------------- | :--------------------------------------------------------------------------------------------------- |
-| URL         | http://localhost:8080        | If you configured the service in Docker Compose with a set container name, you can use that instead. |
-| Record type | pomerium.io/WellKnownIPRange |                                                                                                      |
-| Foreign Key | request.ip                   |                                                                                                      |
+| Field             | Content                      | Notes                                                                                                |
+| :---------------- | :--------------------------- | :--------------------------------------------------------------------------------------------------- |
+| URL               | http://localhost:8080        | If you configured the service in Docker Compose with a set container name, you can use that instead. |
+| Record type       | pomerium.io/WellKnownIPRange |                                                                                                      |
+| Foreign Key       | request.ip                   |                                                                                                      |
+| Polling Min Delay | `1h`                         | Minimum poll interval                                                                                |
+| Polling Max Delay | `4h`                         | Max poll interval                                                                                    |
```

Comment:
nit: if we are going to say minimum we should probably say maximum too.  Or say min / max. Whatever to be consistent.